requirements.txt

```diff
@@ -6,7 +6,7 @@ tensorboard>=2.0
 networkx>=2.0
 quantum-blackbird==0.2.3
 python-dateutil==2.8.0
-thewalrus>=0.11
+http://xanadu-wheels.s3-website-us-east-1.amazonaws.com/thewalrus-0.13.0.dev0+20200519025013-cp36-cp36m-manylinux2010_x86_64.whl
```

Comment:
@ziofil this allows Travis to use the latest version of TheWalrus

Reply:
@antalszava travis cannot install it (see log)